Makefile

```diff
@@ -4,7 +4,8 @@ PYTHON = python3.5
 SHELL = bash
 
 init:
-	git config --global core.hooksPath .githooks
+	find .git/hooks -type l -exec rm {} \;
+	find .githooks -type f -exec ln -sf ../../{} .git/hooks/ \;
```

Comment:
We should also keep the old code so that it works with newer versions of git (on peoples' local machines).

Reply:
this works on both

Reply:
Sorry, misread your description.